### PR TITLE
Add missing ems_id to vnfd template

### PR DESCRIPTION
**Pull request by Ladas**
Add missing ems_id to vnfd template

### CODE REVIEW

**Comment by Ladas** [comment]
@mansam looks like we are missing a spec for this, but if we add validation here https://github.com/ManageIQ/manageiq-providers-openstack/pull/160 it becomes an error

File: app/models/manageiq/providers/openstack/inventory/persister/definitions/cloud_collections.rb [hunk 1]
```diff
@@ -119,6 +119,7 @@ def add_orchestration_stacks(extra_properties = {})
   def add_orchestration_templates
     add_collection(cloud, :orchestration_templates) do |builder|
       builder.add_properties(:model_class => ::OrchestrationTemplate)
+      builder.add_default_values(:ems_id => manager.id)
```
Comment (Ladas) [inline app/models/manageiq/providers/openstack/inventory/persister/definitions/cloud_collections.rb]: hm, now I am not sure if adding it to all templates can cause problems

Comment (Ladas) [reply app/models/manageiq/providers/openstack/inventory/persister/definitions/cloud_collections.rb]: maybe it will be better to add it to parser

Comment (mansam) [reply app/models/manageiq/providers/openstack/inventory/persister/definitions/cloud_collections.rb]: Any further thoughts on this?

Comment (Ladas) [reply app/models/manageiq/providers/openstack/inventory/persister/definitions/cloud_collections.rb]: I am not sure :-)

**Pull request closed without merging**